pom.xml

```
@@ -226,9 +226,7 @@
                             <includes>
                                 javascript-modules-engine/yarn.lock,jahia-test-module/yarn.lock
                             </includes>
-                            <message>[skip ci] [maven-release-plugin] Update yarn.lock files for version
-                                ${project.version}
-                            </message>
+                            <message>ci(release): Prepare release ${project.version}</message>
```

```suggestion
                            <message>chore(release): Prepare release ${project.version}</message>
```

ci is for manual updates of workflow files, chore is more adapted

---

Yep makes sense, thanks!